src/components/Label/index.js

```diff
@@ -6,13 +6,25 @@ import type { Position, Size } from '../../types';
 import { colors, fonts, borders } from '../../assets/styles';
 
 type Props = {
-  tag?: boolean,
   arrow?: Position,
   size?: Size,
+  inverted?: boolean,
+  primary?: boolean,
+  secondary?: boolean,
 };
 
 const getBackgroundColor = R.cond([
-  [R.prop('tag'), R.always(colors.grey)],
+  [
+    R.both(R.propEq('inverted', true), R.propEq('primary', true)),
+    R.always(colors.white),
+  ],
+  [
+    R.both(R.propEq('inverted', true), R.propEq('secondary', true)),
+    R.always(colors.white),
+  ],
+  [R.propEq('inverted', true), R.always(colors.white)],
+  [R.propEq('primary', true), R.always(colors.primary)],
+  [R.propEq('secondary', true), R.always(colors.secondary)],
   [R.T, R.always(colors.grey)],
 ]);
 
```

Comment:
When inverting a primary or secondary label, set the font color to primary or secondary color.